test/e2e/bootstrapper_test.go

```diff
@@ -35,15 +39,30 @@ func TestBootstrapper(t *testing.T) {
 	// TODO: Consider doing the same with kubelet. We can either provide our own or download it from the internet
 	// 		 if we choose to download it, we will have to compare expected vs actual SHA hashes for security reasons
 	ensureIgnitionFileExists(t, ignitionFilePath)
+
+	// Stop the kubelet service so we can remove the kubelet logfile
 	wmcb, err := bootstrapper.NewWinNodeBootstrapper(installDir, ignitionFilePath, kubeletPath)
 	assert.Nilf(t, err, "Could not create WinNodeBootstrapper: %s", err)
+	wmcb.StopAndRemoveServices()
+	wmcb.Disconnect()
+	time.Sleep(5 * time.Second)
+	// Remove the kubelet logfile, so that when we parse it, we are looking at the current run only
+	removeFileIfExists(t, kubeletLogPath)
+
 	// Run the bootstrapper, which will start the kubelet service
+	wmcb, err = bootstrapper.NewWinNodeBootstrapper(installDir, ignitionFilePath, kubeletPath)
```

Comment:
I am thinking of separating test into its own. I feel that we're starting and stopping services as part of this which can make the test disruptive. It's better to run these kind of tests separately and have them tagged only to run under certain scenarios.

Reply:
I dont think being disruptive is a negative. Given that this is an end to end test, which shouldn't be run on servers that need the kubelet up, what is the negative in having a small gap of kubelet service?